src/mgr/FunAction.cc

```diff
@@ -49,10 +48,7 @@ Mgr::FunAction::dump(StoreEntry* entry)
 {
     debugs(16, 5, MYNAME);
     Must(entry != nullptr);
-    if (UsingSmp())
-        storeAppendPrintf(entry, "by kid%d {\n", KidIdentifier);
+
```

Comment:
```suggestion
openKidSection(entry, is_yaml());
```

Reply:
Hmm.. there is a problem here. is_yaml() is a property of Action, which is not accessible from the OBJH callback, which means that right now all FunActions() share the same value here. I'll have to set it up on registration instead, not very clean

Reply:
OBJH functions do not call open/closeKid. FunAction does, for all of them

Reply:
Change request scope: an `openKidSection(entry, is_yaml())` call in Mgr::FunAction::dump().

> Francesco: there is a problem here. is_yaml() is a property of Action, which is not accessible from the OBJH callback

... Alex is trying to be helpful by suggesting a workaround for the described problem (under the assumption that the described problem is about open/closeKid calls in some OBJH callback Francesco is working on) ...

> Francesco: OBJH functions do not call open/closeKid. FunAction does, for all of them

OK, let's start from the beginning then: Please describe the "problem" in more detail. Why do you need to access is_yaml() in an OBJH callback?

N.B. FunAction is an Action so is_yaml() is accessible from FunAction itself.

Reply:
I was thinking about the migration strategy, but if it involves works moving to registering action factory methods, it can work anyway. Changing.